src/test/java/bdd/runner/CucumberTestRunner.java

```diff
@@ -0,0 +1,15 @@
+package bdd.runner;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+
+@CucumberOptions(
+        plugin = "pretty",
+        strict = true,
+        monochrome = true,
+        tags = "@smoke",
```

Comment:
remove this tag

Reply:
Refactored, thank you.
Please see in the following commit.